SECURITY.md

```diff
@@ -0,0 +1,23 @@
+# Hyperledger Security Policy
+
+## Reporting a Security Bug
+
+If you think you have discovered a security issue in any of the Hyperledger
+projects, we'd love to hear from you. We will take all security bugs
+seriously and if confirmed upon investigation we will patch it within a
+reasonable amount of time and release a public security bulletin discussing
+the impact and credit the discoverer.
```

Comment:
This isn't a substantive comment--just a note about verbiage.

We have a bit of a run-on sentence (too many *and*s splicing together too many clauses). It also mixes plural ("security bugs") and singular ("patch it"). Perhaps an improvement would be something like this:

```
We take all security bugs seriously. If we confirm one after investigation,
we will patch it within a reasonable amount of time, and we will release a
public security bulletin that discusses the impact and credits the discoverer.
```

Reply:
That will be round 2.